dvc/command/completion.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import logging
+import os
+
+import shtab
+from dvc.command import choices
```

Comment:
@casperdcl, need a space here, that's why isort is failing.

```suggestion
import shtab

from dvc.command import choices
```

Reply:
sure; but why are my local tests passing? `pre-commit clean && pre-commit run --all-files` passes...

Reply:
just committed that suggestion and got further trolled by this: https://github.com/iterative/dvc/pull/4065

Reply:
@casperdcl, most likely, it's because you haven't added it to `known_third_party`?

Reply:
`seed-isort-config`'s insisting on removing it when I try :thinking:

Reply:
thx found the issue. Had `shtab` on my python path for development in such a way that `isort` didn't think it was third-party.